lambdas/es/indexer/index.py

```diff
@@ -448,6 +451,16 @@ def handler(event, context):
                     size=size,
                     text=text
                 )
+            except botocore.exceptions.ClientError as boto_exc:# pylint: disable=broad-except
+                if key.startswith(FORBIDDEN_PREFIXES) and \
```

Comment:
Any reason not to unify with logic in https://github.com/quiltdata/quilt/blob/74dda6ce5205562c013b42e87e02a7044aad8351/lambdas/es/indexer/index.py#L506?

Reply:
that makes sense, done